app/hooks/solutions/index.ts

```diff
@@ -0,0 +1,91 @@
+import { useMemo, useRef } from 'react';
+import {
+  useInfiniteQuery,
+} from 'react-query';
+import { useSession } from 'next-auth/client';
+
+import flatten from 'lodash/flatten';
+
+// import { ItemProps as RawItemProps } from 'components/gap-analysis/item/component';
+
+import PROJECTS from 'services/projects';
+
+import {
+  UseSolutionsOptionsProps,
+} from './types';
+
+import ITEMS from './mock';
+
+// interface AllItemProps extends RawItemProps {}
+
+export function useSolutions(projectId, options: UseSolutionsOptionsProps = {}) {
+  const placeholderDataRef = useRef({
```

Comment:
Clement told me the other day that React Query has already a way of keeping the previous data. If you check this commit from today https://github.com/Vizzuality/marxan-cloud/commit/0859be0c6122e718be6ec45a38b3c0d349ab5e5b you will see that we don't need the useRef and all of the subsequent uses of it anymore

Reply:
Cool, thanks 👍 
Fixed! ✅